<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Save Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLoad">
       <property name="text">
        <string>Load Image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Selection:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Brush</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blur Brush</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Circle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rectangle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Triangle</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxErase">
         <property name="text">
          <string>Erase</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxFilled">
         <property name="text">
          <string>Filled</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Size:   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderSize">
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="sliderPosition">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Red:   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderRed">
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Green:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderGreen">
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Blue:  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderBlue">
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Alpha:  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderAlpha">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="sliderPosition">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonColorDialog">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Dialog</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>416</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUmint">
    <property name="title">
     <string>Umint</string>
    </property>
   </widget>
   <addaction name="menuUmint"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_5</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_6</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_7</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAlpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_10</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
